tap_bing_ads/__init__.py

```diff
@@ -54,6 +58,39 @@
 
 ARRAY_TYPE_REGEX = r'ArrayOf([A-Za-z0-9]+)'
 
+def should_retry_httperror(exception):
+    """ Return true if exception is required to retry otherwise return false """
+    try:
+        if isinstance(exception, ConnectionError) or isinstance(exception, ssl.SSLError):
+            return True
+        elif isinstance(exception, suds.transport.TransportError) or isinstance(exception, socket.timeout):
+            return True
+        elif type(exception) == URLError:
```

Comment:
Do we need to retry for all URLErrors?

Reply:
Yes. Because here `suds` SDK is used and in some methods, it internally catches many errors including connection reset error, timeout error, etc, and raises URLError of it.